### PR TITLE
suggest the use of maven encrypt password when using maven settings to store credentials

### PR DESCRIPTION
**Pull request by aboullaite**
a small PR updating the `RADME.md`, suggesting to user to use maven encrypt password to encrypt their credentials, if they prefer to use maven settings.

### CODE REVIEW

**Comment by aboullaite** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ Registry credentials can be added to your [Maven settings](https://maven.apache.
 ```
 
 * The `id` field should be the registry server these credentials are for. 
-* We *do not* recommend putting your raw password in `settings.xml`.
+* We *do not* recommend putting your raw password in `settings.xml`. `Maven 2.1` introduced a facility to encrypt passwords in a user's Maven Settings (`~/.m2/settings.xm`l). read more about it [here](https://maven.apache.org/guides/mini/guide-encryption.html)
```
Comment (TadCordle) [inline README.md]: Just some formatting/grammar things to fix: 
* `Maven 2.1` -> Maven 2.1
* Maven Settings -> Maven settings
* `~/.m2/settings.xm`l -> `~/.m2/settings.xml`
* read -> Read
* Should end with a period probably.

Comment (aboullaite) [reply README.md]: Thanks for the remarks @TadCordle :) PR updated

**Comment by coollog** [comment]
Thanks for the contribution!

**Pull request merged**